### PR TITLE
Fix app, test and script logging (#419, #1175) 

### PR DESCRIPTION
**Pull request by hannes-ucsc**

### CODE REVIEW

File: lambdas/service/app.py [hunk 1]
```diff
@@ -67,7 +65,7 @@ def _get_pagination(current_request, entity_type):
     default_sort, default_order = sort_defaults[entity_type]
     pagination = {
         "order": query_params.get('order', default_order),
-        "size": int(query_params.get('size', ENTRIES_PER_PAGE)),
+        "size": int(query_params.get('size', '10')),
```
Comment (jessebrennan) [inline lambdas/service/app.py]: Not critiquing your change, but why 10?

[the remark this answers is not in the document]
Comment (hannes-ucsc) [reply lambdas/service/app.py]: I just inlined the constant and fixed the type. I don't know why the the original author chose 10.

File: scripts/enable_lambdas.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 
 class RedButton:
+
```
[the remark this answers is not in the document]
Comment (hannes-ucsc) [reply scripts/enable_lambdas.py]: One empty line before method defs …

File: test/service/test_response.py [hunk 3]
```diff
@@ -1321,7 +1318,6 @@ def test_ranged_values(self):
 
 
 class TestResponseSummary(WebServiceTestCase):
-
```
[the remark this answers is not in the document]
Comment (hannes-ucsc) [reply test/service/test_response.py]: No empty lines before attributes defs. I think those are the settings my formatter is set to. I'd prefer an empty line after a class declaration but I don't thing PyCharm has a stetting for that.

**Pull request merged**